core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -242,24 +242,24 @@ private void tryStart() {
             logger().info("Starting container with ID: {}", containerId);
             dockerClient.startContainerCmd(containerId).exec();
 
+            logger().info("Container {} is starting: {}", dockerImageName, containerId);
```

Comment:
FYI I remember "print container's output when it fails to start" logic, shouldn't it use `getLogs` now?

Reply:
Good point, will change.